public/app/plugins/panel/geomap/layers/data/utils.ts

```diff
@@ -0,0 +1,84 @@
+import { DataFrame, DataFrameView } from '@grafana/data';
+import { Point } from 'ol/geom';
+import { fromLonLat } from 'ol/proj';
+import { FieldMappingOptions, QueryFormat } from '../../types'
+
+ /**
+   * Function that formats dataframe into a Point[]
+   */
+export function dataFrameToPoints(frame: DataFrame, fieldMapping: FieldMappingOptions, queryFormat: QueryFormat): Point[] {
```

Comment:
This would be a good function to add some simple tests.

Reply:
Yes definitely! A separate PR for dataFrameToPoints logic with tests will be filed soon :D